### PR TITLE
Fixes #24394 - Update about page with link to forums

### PR DESCRIPTION
**Pull request by alexjfisher**
Should I also 'Extract all strings for i18n' in this commit?  It looks like I'd also get unrelated changes.

### CODE REVIEW

File: app/views/about/index.html.erb [hunk 1]
```diff
@@ -130,12 +130,8 @@
...
       <h6><%= _("IRC") %></h6>
       <p><%= (_("You can find The Foreman on the %{freenode} (irc.freenode.net) network.  For general support, please visit #theforeman and for development specific related chat, please visit #theforeman-dev.") % {:freenode => link_to("freenode", "http://www.freenode.net", :rel => "external")}).html_safe  %></p>
-      <h6><%= _("Mailing lists") %></h6>
-      <p><%= _("Mailing lists are available via Google Groups. Much like IRC, we have a general users (support, Q/A, etc) lists and a development list:") %></p>
-      <ul>
-        <li><%= link_to _("Foreman Users"), "http://groups.google.com/group/foreman-users", :rel => "external" %></li>
-        <li><%= link_to _("Foreman Developers"), "http://groups.google.com/group/foreman-dev", :rel => "external" %></li>
-      </ul>
+      <h6><%= _("Forums") %></h6>
+      <p><%= (_("You can use the %{s} to ask for help or advice, participate in discussions around challenges or ideas you have with Foreman.") % {:s => link_to( _('community forums'), "https://community.theforeman.org/", :rel => "external")}).html_safe %></p>
```
Comment (tbrisker) [inline app/views/about/index.html.erb]: For just one parameter, you can use `%s`  for the parameter, or if you want to give the translators context name the parameter `community_forum` instead (similar to freenode above)

**Review by tbrisker — CHANGES_REQUESTED** [review]
Thanks @alexjfisher! left a minor comment inline

File: app/views/about/index.html.erb (hunk 1 above)
Comment (alexjfisher) [reply app/views/about/index.html.erb]: Fixed

**Comment by tbrisker** [comment]
ok to test (this kicks off ci just to make sure we don't happen to have some test expecting the old string)

**Review by tbrisker — DISMISSED** [review]

**Review by ekohl — DISMISSED** [review]

**Comment by tbrisker** [comment]
@alexjfisher - test failure seems to be related, looks like the integration test for the about page also needs updating

**Comment by alexjfisher** [comment]
@tbrisker Sorry about that.  All fixed now.

**Review by boaz0 — APPROVED** [review]
LGTM :+1:

**Review by tbrisker — APPROVED** [review]
Thanks @alexjfisher !

**Comment by tbrisker** [comment]
cherry-picked to 1.19 as  151d5c006

**Pull request merged**